SURVEY.md

```diff
@@ -48,11 +48,20 @@ How do we want people to indicate their language?
     add as many as appropriate.
 
 ### Racial Identification
-Allowing people to check all labels with which they identify would be good.
+    Do you identify as a person of color? (e.g. African, Latino/Hispanic, Asian, South Asian, Native American, Middle Eastern, Pacific Islander [1])
 
-    Choose the racial identities (phrasing!) with which you identify
+    () Yes
+    () No
+
+    With which ethnicity do you most identify (from Information is Beautiful [2])?
 
-    - combo box of words (tbd)
+    () White
```

Comment:
I'd like to hear from some people of color before this gets merged - or we can at least change the () to [] to let people choose more than one without the 'multi' box.